storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="persistent-storage-csi-smb-cifs"]
+= SMB CIFS CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: persistent-storage-csi-smb-cifs
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Server Message Block (SMB) protocol Common Internet File System (CIFS) dialect.
+
+:FeatureName: SMB CIFS CSI Driver Operator
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+After installing the SMB CIFS CSI Driver Operator, {product-title} installs the SMB CIFS CSI Operator and the SMB CIFS CSI driver by default in the `openshift-cluster-csi-drivers` namespace. This allows the SMB CIFS CSI Driver Operator to create CSI-provisioned PVs that mount to SMB CIFS assets.
```

Comment:
After Installing the SMB CIFS CSI Driver Operator we need to do the similar thing to create the smb clustercsidriver to trigger the csidriver install.(ref: https://docs.openshift.com/container-platform/4.15/storage/container_storage_interface/persistent-storage-csi-aws-efs.html#persistent-storage-csi-efs-driver-install_persistent-storage-csi-aws-efs)
```console
apiVersion: operator.openshift.io/v1
kind: ClusterCSIDriver
metadata:
    name: smb.csi.k8s.io
spec:
  managementState: Managed

```

Reply:
For example, for GCP Filestore, this comes as last step in "Installing the GCP Filestore CSI Driver Operator" immediately before "Creating a storage class for GCP Filestore Storage":

> 4. Install the GCP Filestore CSI Driver:
> a. Click administration → CustomResourceDefinitions → ClusterCSIDriver.
> b. On the Instances tab, click Create ClusterCSIDriver.

(https://docs.openshift.com/container-platform/4.15/storage/container_storage_interface/persistent-storage-csi-google-cloud-file.html)

We do need similar wording for SBM, imho.